app/src/main/java/org/sopt/sample/presentation/types/SoptPartType.kt

```diff
@@ -0,0 +1,12 @@
+package org.sopt.sample.presentation.types
+
+import org.sopt.sample.R
+
+enum class SoptPartType(val strRes: Int) {
```

Comment:
```suggestion
enum class SoptPartType(@StringRes val strRes: Int) {
```

resource id 같은 경우는 메타 어노테이션 명시해주시면 좋을 것 같습니다.